### ISSUE TITLE
Is it correct that clusterID is set to targetID which leads to effectively no clusters?

### ISSUE THREAD

**Issue by hubtub2** [DIDSR/mitoticFigureCounts#2]
Hello,

I am trying to use this method / code for a similar setting. Thank you for this work, I think this is exactly what we need.
While running and analyzing your code (05_doMRMCaucORcluster.R) I stumbled upon the following:

In line 88 you create the new _caseID_ column from the _targetID_ column:
`df.convert$caseID <- df.convert$targetID`

Later in line 53 you call doAUCcluster() with the parameter
 `clusterID  = df.merge$caseID.x,`

That means the _clusterID_ is set to the original column _targetID_.
Looking at the CSV data in _dfClassify.csv_ I see, that each row has its own _targetID_ (the value repeats only for the five modalities).

Debugging to line 53 shows that **there are effectively no clusters** (or all clusters have length 1):
```
length(unique(df.merge$caseID.x))
[1] 155
```

Reading the original documentation in clusteredROC_help.pdf, it says that in the original MRA example the _clusterID_ is the _patientID_.

I wonder why you use _targetID_ for the _clusterID_ parameter and not the locationID/roiID or wsiName?

Thank you again for the great work and I assume it must be some misunderstanding from my side.

Best,
Stefan

**Comment by brandon-gallas** [comment]
I've been preparing for and attending a meeting this week. I plan to investigate your question and get back to you next week.

Thanks

**Comment by brandon-gallas** [comment]
You have identified a bug.

Line 53 should read
`clusterID  = df.merge$wsiName.x`

**Thank you for your patience, and thank you for pointing out this bug.**
* I have renamed the original results from _05_doMRMCaucORcluster.R_ to be `aucMRMCclusterOriginal`.
* I have made the change to clusterID in these files: _05_doMRMCaucORcluster.R_, _05_doMRMCtnfORcluster.R_, _05_doMRMCtpfORcluster.R_
* I have saved the updated results to `aucMRMCcluster`. There are differences compared to the original results, but nothing that changes the message from the paper.
